_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,124 @@
+---
+title: How to update to the newest version of CIS AWS Foundations Benchmark
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update to the 1.3.0 version of the CIS AWS Foundations Benchmark
+tags: ["aws", "security", "compliance"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-to-cis-13/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+
+This guide will walk you through the upgrade process from the 1.2.0 to the 1.3.0 version of the CIS AWS Foundations Benchmark.
+If your infrastructure is already compliant with the 1.2.0 version of the Benchmark, and you are looking to upgrade to 1.3,
+this guide is for you.
+
+=== What you'll learn in this guide
+
+This guide consists of two main sections:
+
+<<core_concepts>>::
+  An overview of the CIS AWS Foundations Benchmark 1.3.0 and why it is important to update your code for compatibility.
+
+<<deployment_walkthrough>>::
+  The steps you need to take to update your code to be compliant with CIS AWS 1.3.0. It includes a
+  <<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+  tag is compatible with CIS AWS 1.3.0, as well as the manuals step you need to perform to achieve said compliance.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
```

Comment:
Could you add a section under Core Concepts that summarizes what's new in the 1.3 version of the benchmark? That is, the new requirements added after 1.2? It could be a bullet point list.